system-variables.md

```diff
@@ -1,129 +1,690 @@
 ---
-title: The System Variables
-summary: Learn how to use the system variables in TiDB.
+title: System Variables
+summary: Use system variables to optimize performance or alter running behavior.
 category: reference
-aliases: ['/docs/dev/system-variables/','/docs/dev/reference/configuration/tidb-server/mysql-variables/']
+aliases: ['/docs/dev/tidb-specific-system-variables/','/docs/dev/reference/configuration/tidb-server/tidb-specific-variables/', '/tidb/dev/tidb-specific-system-variables']
 ---
 
-# The System Variables
+# System Variables
 
-The system variables in MySQL are the system parameters that modify the operation of the database runtime. These variables have two types of scope, Global Scope and Session Scope. TiDB supports all the system variables in MySQL 5.7. Most of the variables are only supported for compatibility and do not affect the runtime behaviors.
+TiDB system variables behavior similar to MySQL, in that settings may apply on a `SESSION`, `GLOBAL` or both `SESSION` and `GLOBAL` scope. Changes to `GLOBAL` scoped variables **only apply** to new connections to TiDB. Variables can be set with the [`SET` statement](/sql-statements/sql-statement-set-variable.md) on a per-session or global basis:
 
-## Set the system variables
+```sql
+# These two identical statements change a session variable
+SET tidb_distsql_scan_concurrency = 10;
+SET SESSION tidb_distsql_scan_concurrency = 10;
 
-You can use the [`SET`](/sql-statements/sql-statement-set-variable.md) statement to change the value of the system variables. Before you change, consider the scope of the variable. For more information, see [MySQL Dynamic System Variables](https://dev.mysql.com/doc/refman/5.7/en/dynamic-system-variables.html).
+# These two identical statements change a global variable
+SET @@global.tidb_distsql_scan_concurrency = 10;
+SET  GLOBAL tidb_distsql_scan_concurrency = 10;
+```
 
-### Set Global variables
+> **Note:**
+>
+> TiDB differs from MySQL in that `GLOBAL` scoped variables **persist** through TiDB server restarts. Changes will also be propagated to other TiDB servers every 2 seconds [TiDB #14531](https://github.com/pingcap/tidb/issues/14531).
+> Additionally, TiDB presents several MySQL variables from MySQL 5.7 as both readable and settable. This is required for compatibility, since it is common for both applications and connectors to read MySQL variables. For example: JDBC connectors both read and set query cache settings, despite not relying on the behavior.
 
-Add the `GLOBAL` keyword before the variable or use `@@global.` as the modifier:
+## Variable Reference
```

Comment:
Hello   @nullnotnil ，I think this pr means fix the vars in the issue https://github.com/pingcap/docs/issues/3155 , but i did not see them in this pr ?

And some var is already in Chinese document，PTAL    https://docs.pingcap.com/zh/tidb/stable/tidb-specific-system-variables#tidb_enable_index_merge-%E4%BB%8E-v40-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5

Reply:
See the not in the description:
> In the process of creating this PR, I discovered that several sysvars are undocumented. I have created #3155 to handle separately.

The chinese version looks in better shape than the english document, but the variables are still not in alphabetical order.